web-ui/package.json

```diff
@@ -3,12 +3,14 @@
   "license": "MIT",
   "scripts": {
     "ng": "ng",
+    "ng-high-memory": "node --max_old_space_size=8192 ./node_modules/@angular/cli/bin/ng",
     "start": "npm run build-pre && npm run serve & npm run test",
     "start:live": "npm run build-pre && npm run serve:live & npm run test",
     "serve": "ng serve --proxy-config proxy.conf.json",
     "serve:live": "ng serve --proxy-config proxy.live.conf.json",
     "serve:poll": "ng serve --poll=2000 --host 0.0.0.0 --proxy-config proxy.conf.json",
     "build": "npm run build-pre && ng build --prod --base-href=/ng/ --output-path=../ng",
+    "build-prod": "npm run build-pre && npm run ng-high-memory build --prod --base-href=/ng/ --output-path=../ng",
```

Comment:
I think we could set this to use for `"build"` instead rather than having a separate `"build-prod"`. It's reasonable to assume this problem also exists on the developer's local machine, and this saves having to figure that one out yourself again.

Reply:
Yeah that makes sense.